integration-cli/docker_cli_build_test.go

```diff
@@ -4694,3 +4694,51 @@ func TestBuildDockerfileOutsideContext(t *testing.T) {
 
 	logDone("build - Dockerfile outside context")
 }
+
+func TestBuildMissingArgs(t *testing.T) {
+	// test to make sure these cmds w/o any args will generate an error
+	// Notice some commands are missing because its ok for them to
+	// not have any args - like: CMD, RUN, ENTRYPOINT
```

Comment:
@duglin I'm pretty sure all commands need an argument. Even your main PR description only mentions *future* instructions.

Also should we keep in the parser code a map like: `var CmdNArg = map[string]int { "ADD": 1, ... }` or `map[int][]string { 1: {"ADD", "COPY", "ENV", ...}, }` ?